### PR TITLE
File added:  Vijaya Boostrap assignments 5 and 6

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

**Review by shubhamdalal49 — APPROVED** [review]

File: vijaya/Boostrap/assignment6/assign6login.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    <label for="">Email</label>
+                    <input type="email" class="form-control" placeholder="Email">
+                </div>
+                <div class="form-group d-flex">
+                    <label for="">Password</label>
+                    <input type="password" class="form-control" placeholder="Password">
+                </div>
+                <button class="btn btn-primary" type="submit">Submit</button>
+            </div>
+
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment6/assign6login.html]: remove extra line

File: vijaya/Boostrap/assignment6/assign6login.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                </div>
+                <div class="form-group d-flex">
+                    <label for="">Password</label>
+                    <input type="password" class="form-control" placeholder="Password">
+                </div>
+                <button class="btn btn-primary" type="submit">Submit</button>
+            </div>
+            
+        </form>
+    </div>
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment6/assign6login.html]: check indentation

File: vijaya/Boostrap/assignment6/assign6login.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment6/assign6login.html]: remove extra line

File: vijaya/Boostrap/assignment6/assign6academic.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                        <input class="class="form-check-input type="checkbox"><span>&nbsp;Html &nbsp;</span>
+                        <input class="class="form-check-input type="checkbox"><span>&nbsp;CSS &nbsp;</span>
+                        <input class="class="form-check-input type="checkbox"><span>&nbsp;Java &nbsp;</span>
+                        <input class="class="form-check-input type="checkbox"><span>&nbsp;Python</span>
+                    </div>
+                </div> 
+                <button class="btn btn-primary" type="submit">Submit</button>
+            </div>
+        </form>
+    </div>
```
Comment (nileshp1122) [inline vijaya/Boostrap/assignment6/assign6academic.html]: check indentation

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make changes asap

**Comment by vijaya1811** [comment]
> make changes asap

yes sir I done changes and pushed again please review

File: vijaya/Boostrap/assignment6/assign6academic.html (hunk 4 above)
Comment (vijaya1811) [reply vijaya/Boostrap/assignment6/assign6academic.html]: yes done please review

**Review by nileshp1122 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**